Lib/sqlite3/__main__.py

```diff
@@ -7,6 +13,14 @@
 
 
 def execute(c, sql, suppress_errors=True):
+    """Helper that wraps execution of SQL code.
+
+    This is used both by the REPL and by direct execution from the CLI.
+
+    'c' may be a cursor or a connection.
+    'sql' is the SQL string to execute
```

Comment:
```suggestion
    'sql' is the SQL string to execute.
```

Trivial nit, per PEP 257 and consistency